staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -177,11 +176,21 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		5*time.Minute, // this is effectively used as a refresh interval right now.  Might want to do something nicer later on.
 	)
 
+	var certKeyContentProvider *dynamiccertificates.DynamicCertKeyPairContent
+	if len(c.ExtraConfig.ProxyClientCertFile) > 0 && len(c.ExtraConfig.ProxyClientKeyFile) > 0 {
```

Comment:
if there isn't a client cert/key pair, then the front proxy fails and we should return an error, right?